pymc/tests/test_variational_inference.py

```diff
@@ -26,18 +26,16 @@
 from pymc.aesaraf import intX
 from pymc.tests import models
 from pymc.tests.helpers import not_raises
-from pymc.variational import flows, opvi
+from pymc.variational import opvi
 from pymc.variational.approximations import (
     Empirical,
     EmpiricalGroup,
     FullRank,
     FullRankGroup,
     MeanField,
     MeanFieldGroup,
-    NormalizingFlow,
-    NormalizingFlowGroup,
 )
-from pymc.variational.inference import ADVI, ASVGD, NFVI, SVGD, FullRankADVI, fit
```

Comment:
`fit` should be also tested. What is the purpose of removing it?

Reply:
`+ def ignore_not_implemented_inference(...)` below should also be removed

Reply:
I removed fit to avoid a pre-commit error; I'll reintroduce it once I repair the test that uses the `fit`.